package/android/cpp/jni/include/JniSkiaBaseView.h

```diff
@@ -7,8 +7,8 @@
 #include <jni.h>
 #include <jsi/jsi.h>
 
-#include <JniSkiaManager.h>
-#include <RNSkAndroidView.h>
+#include "JniSkiaManager.h"
```

Comment:
For this part, is it possible to make a separate PR?

Reply:
For which part? Removing the header? Or changing `<` to `"`?